grafana-plugin/src/models/alertgroup/alertgroup.ts

```diff
@@ -298,6 +298,12 @@ export class AlertGroupStore extends BaseStore {
     });
   }
 
+  @action
```

Comment:
actually we don't need `action` here, " An action is any piece of code that modifies the state."